assets/scss/_custom.scss

```diff
@@ -648,6 +648,9 @@ body.td-documentation {
 }
 
 #announcement {
+  // default background is blue; overrides are possible
+  color: #fff;
```

Comment:
nit: Not sure whether `#fff` or `white` is better.

Reply:
I like hex codes for the greater degree of specificity they permit. The issue is certainly a conversation we can have as a community. But for now--

/lgtm